src/main/java/io/appform/dropwizard/actors/actor/MessageProperties.java

```diff
@@ -0,0 +1,13 @@
+package io.appform.dropwizard.actors.actor;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class MessageProperties {
+
+    private boolean isRedelivered;
```

Comment:
maybe name it just `redelivered`. Getter would itself become `isRedelivered`

Reply:
renamed.